### DISCUSSION TITLE
Golang not detected in a repo

### DISCUSSION THREAD

**Discussion by Manish-Giri** [github-linguist/linguist#6580] [Classification] [answered]
## Describe the bug

I have this [repo](https://github.com/Manish-Giri/leetcode) which used to contain only Java and Python code up until recently. Both languages were correctly classified.

About a week ago I added some Go code to the repo, in this [commit](https://github.com/Manish-Giri/Leetcode/commit/0141e817f8a9743072fa48105a665fdc857dd439), but Go was not detected as one of the languages, after the commit. Not only that, both Python and Go are now classified as "Other" while Java is the only language that shows up.

<img width="325" alt="Screenshot 2023-10-12 at 11 00 39 PM" src="https://github.com/github-linguist/linguist/assets/11348778/3a9d5817-7243-4699-82f7-bfa9d333a5c9">

### Expected behaviour

All languages should be correctly classified.

**Comment by lildude** [comment, marked as answer]
> All languages should be correctly classified.

They are, however you have so much Java by bytes of code, that it makes the other languages almost insignificant in comparison as you can see from the 99.6% vs 0.4% breakdown, so GitHub doesn't show those other languages. IIRC, the cut off is something like at least one or two percent per language. Linguist has no control over this decision or percentage.

**Reply by Manish-Giri** [reply]
All right, thank you.
